torch/csrc/jit/passes/quantization.cpp

```diff
@@ -967,12 +994,27 @@ graph(%self, %a_dequant, %w_scale, %w_zero_point, %w_dtype):
           toIValue(match_vmap.at(vmap.at("w_dtype"))).value().toScalarType();
       auto w = module.get_parameter("weight").variable_data();
       auto w_quant = at::quantize_per_tensor(w, w_scale, w_zero_point, w_dtype);
-      auto b = module.get_parameter("bias").variable_data();
-      auto wrapper = wrapper_module.clone();
-      auto set_weight_bias = wrapper.get_method("set_weight_bias");
+      c10::optional<at::Tensor> b = c10::nullopt;
+      if (module.find_parameter("bias")) {
+        b = module.get_parameter("bias").variable_data();
+      }
+      auto wrapper_module = packed_params_module.clone();
+      auto set_weight_bias = wrapper_module.get_method("set_weight_bias");
+      if (is_conv) {
+        auto stride = toIntList(match_vmap.at(vmap.at("stride")));
+        auto padding = toIntList(match_vmap.at(vmap.at("padding")));
+        auto dilation = toIntList(match_vmap.at(vmap.at("dilation")));
+        auto groups = toIValue(match_vmap.at(vmap.at("groups")));
+        auto set_conv_params = wrapper_module.get_method("set_conv_params");
+        if (!stride || !padding || !dilation) {
+          TORCH_WARN("Non-constant stride/padding/dilation");
+          continue;
+        }
+        set_conv_params(std::vector<IValue>{stride.value(), padding.value(), dilation.value(), groups.value()});
+      }
       set_weight_bias(std::vector<IValue>{IValue(w_quant), IValue(b)});
       // TODO: we need to make sure this name is unique
-      module.register_module("_packed_linear_weight_bias", wrapper);
+      module.register_module("_packed_weight_bias", wrapper_module);
```

Comment:
This will not work if a module has both `linear` and `conv2d` (or several `conv`s).

Reply:
yeah see the comment above, I'm planning to do this in a separate PR